docs/wiki/[Variables]-subscription_id_management.md

```diff
@@ -1 +1,31 @@
-_coming soon_
\ No newline at end of file
+## Overview 
+[**subscription_id_managmement**] `string` (optional)
+
+If specified, identifies the Platform subscription for \"Management\" for resource deployment and correct placement in the Management Group hierarchy.
+
+## Default value
+```hcl
+{}
+```
+
+
+
+## Validation
+The subscription can be a passed in string that is 36 characters long containing alphanumeric characters, a string passed in through the variable "subscription_id_management", or an empty white string in the event that it's pulled in from the provider 
+ ```hcl
+  validation {
+    condition     = can(regex("^[a-z0-9-]{36}$", var.subscription_id_management)) || var.subscription_id_management == ""
+    error_message = "Value must be a valid Subscription ID (GUID)."
+  }
+}
+```
+## Usage
+To assign a subscription id to the management subscription. This allows the flexibility to have a unique subscription for the management group if required. 
```

Comment:
We don't assign a Subscription ID to the Management Subscription, rather we specify the Subscription ID of the Management Subscription.

This is to ensure the data model can generate the correct values when a deployment is split into multiple pipelines, not all of which may actually deploy the management resources but may need the values for other resources.